### PR TITLE
Fixes ahelp trimming

### PR DESCRIPTION
**Pull request by AnturK**
Where am i ? This weird dream again. Deja vu ?

### CODE REVIEW

**Comment by vuonojenmustaturska** [comment]
https://www.youtube.com/watch?v=4ezd24krgiQ

**Review by Cyberboss — DISMISSED** [review]

File: code/modules/admin/verbs/adminpm.dm [hunk 1]
```diff
@@ -111,7 +111,9 @@
 		//get message text, limit it's length.and clean/escape html
 		if(!msg)
 			msg = input(src,"Message:", "Private message to [key_name(recipient, 0, 0)]") as text|null
-
+
```
Comment (RemieRichards) [inline code/modules/admin/verbs/adminpm.dm]: ironically, this PR needs trimming itself.

Comment (ShizCalev) [inline code/modules/admin/verbs/adminpm.dm]: tabs

File: code/modules/admin/verbs/adminpm.dm [hunk 2]
```diff
@@ -111,7 +111,9 @@
 		//get message text, limit it's length.and clean/escape html
 		if(!msg)
 			msg = input(src,"Message:", "Private message to [key_name(recipient, 0, 0)]") as text|null
-
+			
+			msg = trim(msg)
+
```
Comment (ShizCalev) [inline code/modules/admin/verbs/adminpm.dm]: tabs

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Pull request merged**